composer.json

```diff
@@ -19,7 +19,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^4.5 || ^5.4",
-        "guzzlehttp/psr7": "^1.3"
+        "guzzlehttp/psr7": "^1.3",
+        "php-http/message": "^1.0"
```

Comment:
No, I do not think so. But the client-common:1.1 require php-http/message. But when we where doing the "prefer-lowest" we used client-common:1.0 and there was no php-http/message. So discovery failed.